javascript/packages/cli/src/actions/spawn.ts

```diff
@@ -93,8 +94,10 @@ export async function spawn(
     force,
     inCI,
     silent: false,
+    setGlobalNetwork,
   };
   const network = await start(creds, config, options);
   network.showNetworkInfo(config.settings?.provider);
-  return network;
+  // keep the process running
+  await new Promise(() => setInterval(() => {}, 1000));
```

Comment:
I think we don't need a promise here.
```suggestion
setInterval(() => {}, 1000);
```